auth-service/src/main/kotlin/uk/gov/communities/delta/auth/Application.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.communities.delta.auth
+
+import io.ktor.server.application.*
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+import uk.gov.communities.delta.auth.plugins.configureMonitoring
+import uk.gov.communities.delta.auth.plugins.configureRouting
+import uk.gov.communities.delta.auth.plugins.configureSecurity
+import uk.gov.communities.delta.auth.plugins.configureSerialization
+
+fun main() {
```

Comment:
What did you use to set up the project? Intellij's new ktor project?

Reply:
Yep